### DISCUSSION TITLE
using CTkImage and Pillow for button image creates random border

### DISCUSSION THREAD

**Discussion by ThomsonW04** [TomSchimansky/CustomTkinter#721] [General]
![image](https://user-images.githubusercontent.com/64808819/205108789-827922b8-9233-40cd-aed4-732e627505b9.png)
you would expect this to be touching the edge, instead it overflows, highlighting the button also shows there is a border
![image](https://user-images.githubusercontent.com/64808819/205108993-d279b049-c4e2-49d4-9a03-33af4fd835dd.png)
Gif showing border: https://gyazo.com/4b8f1d59561b501a8081d78ef2d68755

**Comment by TomSchimansky** [comment]
I don't know what you mean, can you post the original image and a bigger screenshot of the button?

**Reply by ThomsonW04** [reply]
Yeah sure, I will when I'm next at my computer tomorrow after I finish work.
